### PR TITLE
update photo gallery

### PR DESCRIPTION
**Pull request by alisa911**

### CODE REVIEW

**Comment by alisa911** [comment]
https://github.com/osmandapp/OsmAnd/issues/17949

File: map/src/menu/search/ImageItem.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        ctx.setSelectedPhotoInd(index);
+    }, []);
+
+    return (
+        <ImageListItem ref={ref} onClick={() => handlePhotoClick(index)}>
+            {!isLoaded && <Skeleton variant="rectangular" width="100%" />}
+            {inView && (
+                <img
+                    src={`${WIKI_IMAGE_BASE_URL}${photo.properties.imageTitle}?width=300`}
+                    alt={`thumbnail-${index}`}
```
Comment (RZR-UA) [inline map/src/menu/search/ImageItem.jsx:24]: Are you sure that user will be interested to see "thumbnail-123" on mouseover?

**Review by RZR-UA — APPROVED** [review]

**Review by RZR-UA — APPROVED** [review]

File: map/src/menu/search/PhotosModal.jsx [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+    };
+
+    const handleBack = () => {
+        ctx.setSelectedPhotoInd(Math.max(activeStep - 1, 0));
+    };
+
+    useEffect(() => {
+        const handleKeyDown = (event) => {
+            if (event.key === 'ArrowRight') {
+                handleNext();
```
Comment (RZR-UA) [inline map/src/menu/search/PhotosModal.jsx:85]: Please add "Space" key as Next.

File: map/src/menu/search/PhotosModal.jsx [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+            } else if (event.key === 'ArrowLeft') {
```
Comment (RZR-UA) [inline map/src/menu/search/PhotosModal.jsx:86]: Please add "Backspace" key as Back.

File: map/src/setupProxy.js [hunk 4]
```diff
@@ -46,4 +47,5 @@ module.exports = function (app) {
     // app.use('/weather/', weather); // defined-by-env
     // app.use('/search/', others); // actually /routing/search
     app.use('/online-routing-providers.json', others); // osrm-providers
+    app.use('/wiki/', wikimediaProxy);
```
Comment (RZR-UA) [inline map/src/setupProxy.js:50]: Please add comment:

// production uses nginx server-docz-include.conf

**Review by RZR-UA — APPROVED** [review]

**Pull request merged**